TransformationSystem/Service/TransformationManagerHandler.py

```diff
@@ -136,33 +136,35 @@ def export_addTaskForTransformation( self, transName, lfns = [], se = 'Unknown'
     res = database.addTaskForTransformation( transName, lfns = lfns, se = se )
     return self._parseRes( res )
 
-  types_setFileStatusForTransformation = [transTypes, [basestring, dict]]
-  def export_setFileStatusForTransformation( self, transName, dictOfNewFilesStatus, lfns = [], force = False ):
+  def _wasFileInError( self, newStatus, currentStatus ):
```

Comment:
You defined this method also in the TransformationClient, exactly the same. You can have only one definition, and it doesn't need to be a method, it can be a simple function.